app/views/suppliers/items/edit.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2>Edit Item</h2>
+<%= render partial: "./partials/edit_item" %>
```

Comment:
Just to confirm, this is a partial because the super-admin will also be using it?